### PR TITLE
[zh-cn] Fix links in the blog Kubernetes Removals and Deprecations In 1.24

### PR DESCRIPTION
**Pull request by mengjiao-liu**
Co-authored-by: fusida <gujiu.fsd@alibaba-inc.com>

Previous PR: #35730

Preview page: https://deploy-preview-35777--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2022/04/07/upcoming-changes-in-kubernetes-1-24/

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ Docker Engine dependencies. Before upgrading to v1.24, you decide to either rema
 你可以[从查明节点上所使用的容器运行时](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/)开始。
 如果你的节点使用 dockershim，则还有其他可能的 Docker Engine 依赖项，
 例如 Pod 或执行 Docker 命令的第三方工具或 Docker 配置文件中的私有注册表。
-你可以按照[检查弃用 Dockershim 对你的影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
+你可以按照[检查移除 Dockershim 对你的影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
 的指南来查看可能的 Docker 引擎依赖项。在升级到 1.24 版本之前， 你决定要么继续使用 Docker Engine 并
 [将 Docker Engine 节点从 dockershim 迁移到 cri-dockerd](/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)，
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:221]:
```suggestion
[将 Docker Engine 节点从 dockershim 迁移到 cri-dockerd](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)，
```

File: content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 2]
```diff
@@ -216,7 +216,7 @@ Docker Engine dependencies. Before upgrading to v1.24, you decide to either rema
 你可以[从查明节点上所使用的容器运行时](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/)开始。
 如果你的节点使用 dockershim，则还有其他可能的 Docker Engine 依赖项，
 例如 Pod 或执行 Docker 命令的第三方工具或 Docker 配置文件中的私有注册表。
-你可以按照[检查弃用 Dockershim 对你的影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
+你可以按照[检查移除 Dockershim 对你的影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:219]:
```suggestion
你可以按照[检查移除 Dockershim 是否对你有影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
```
Do we need to follow the linked title after click? 😃

File: content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md (hunk 1 above)
Comment (mengjiao-liu) [reply content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md]: oh.. Thanks for catch!

File: content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md (hunk 2 above)
Comment (mengjiao-liu) [reply content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md]: yes. Updated.

File: content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 3]
```diff
@@ -216,9 +216,9 @@ Docker Engine dependencies. Before upgrading to v1.24, you decide to either rema
 你可以[从查明节点上所使用的容器运行时](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/)开始。
 如果你的节点使用 dockershim，则还有其他可能的 Docker Engine 依赖项，
 例如 Pod 或执行 Docker 命令的第三方工具或 Docker 配置文件中的私有注册表。
-你可以按照[检查弃用 Dockershim 对你的影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
+你可以按照[检查移除 Dockershim 是否对你有影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
 的指南来查看可能的 Docker 引擎依赖项。在升级到 1.24 版本之前， 你决定要么继续使用 Docker Engine 并
-[将 Docker Engine 节点从 dockershim 迁移到 cri-dockerd](/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)，
+[将 Docker Engine 节点从 dockershim 迁移到 cri-dockerd](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)，
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:221]:
```suggestion
[将 Docker Engine 节点从 dockershim 迁移到 cri-dockerd](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd/)，
```

File: content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 4]
```diff
@@ -216,9 +216,9 @@ Docker Engine dependencies. Before upgrading to v1.24, you decide to either rema
 你可以[从查明节点上所使用的容器运行时](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/)开始。
 如果你的节点使用 dockershim，则还有其他可能的 Docker Engine 依赖项，
 例如 Pod 或执行 Docker 命令的第三方工具或 Docker 配置文件中的私有注册表。
-你可以按照[检查弃用 Dockershim 对你的影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
+你可以按照[检查移除 Dockershim 是否对你有影响](/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/)
 的指南来查看可能的 Docker 引擎依赖项。在升级到 1.24 版本之前， 你决定要么继续使用 Docker Engine 并
```
Comment (Sea-n) [inline content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:220]: Unnecessary space.

And mix of "Docker 引擎" and "Docker Engine".

```suggestion
的指南来查看可能的 Docker 引擎依赖项。在升级到 1.24 版本之前，你决定要么继续使用 Docker Engine 并
```

File: content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md (hunk 3 above)
Comment (mengjiao-liu) [reply content/zh-cn/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md]: whoops! I missed that!

**Pull request merged**